### ISSUE TITLE
Feedback

### ISSUE THREAD

**Issue by denaefordrobin** [gvwilson/10-newcomers#25]
Rule 1:
- Capitalize + reword title: “Rule 1: have and enforce a code of conduct.” —> “Rule 1: Create, Evolve,  and Enforce a Code of Conduct”
- First sentence says codify: What does codify in this context mean? We don't want them to be 'coded' we want them to be clear and easy to interpret. Maybe we want to say ‘present’.
- I think we should reduce content on how to enforce a code of conduct and offer an example of how to create/evolve a code of conduct with (structured)community feedback. 
    - Community Feedback: cc// Procedural Justice literature: e.g., Have a community forum/channel where users can provide feedback on the rules. If the communities have the ability to give feedback on the rules in some way this help users perceive the rules as more fair.
        - This will dismantle the idea that some unknown authoritative entity made the rules and users are just living them. Ultimately, this will encourage the idea that contributors at all stages have the ability to give feedback. Tyler et al. have found that giving group members a voice in the process can influence how fair the process may be[1].
    - Evolve: Adopt a process to digest feedback and acknowledge it via updates to the Code of Conduct as necessary.
- Fix: “quickly recognize the code as meaningless.” —> “quickly recognize the code of conduct as meaningless.”

Rule 2: Iooks good.
Rule 3: looks good.
Rule 4: I really like this rule
Rule 5: I agree with Igor. In it’s current position in the list it feels out of place. Perhaps it will have a better place before ‘be welcoming’.
- concepts should be italicized and capitalized(do not add quotes): “Intimidating Community Size” + Fear of Negative Feedback”

Rule 6: looks good.
Rule 7: looks good.
Rule 8: looks good.
Rule 9:
- Capitalize Title:  “Rule 9: make it..”—> “Rule 9: Make it…”
- Really like sentence starting with  “By treating the process of getting involved..”
Rule 10: looks good. nice ending sentence.

References
[1]
@article{tyler1996understanding,
  title={Understanding why the justice of group procedures matters: A test of the psychological dynamics of the group-value model.},
  author={Tyler, Tom and Degoey, Peter and Smith, Heather},
  journal={Journal of personality and social psychology},
  volume={70},
  number={5},
  pages={913},
  year={1996},
  publisher={American Psychological Association}
}

**Comment by gvwilson** [comment]
I agree the rules should be capitalized, but unfortunately that's not PLOS house style.

**Comment by gvwilson** [comment]
@denaefordrobin can you please suggest wording for "reduce content on how to enforce a code of conduct"? A lot of the discussions I've seen/heard in the last two years have emphasized that a CoC is (worse than) useless without enforcement, and several people have told me that picking a CoC is a much more productive route than having people try to come up with one (just as adopting a license is less painful than writing one).

Alternatively, is it useful to forward ref from here to Rule 2, and mention "mechanism for changing the rules" as part of making governance explicit?

**Comment by gvwilson** [comment]
I think that capitalizing Intimidating Community Size and Fear of Negative Feedback will look odd in context - I recognize these are categories from the paper cited, but that information isn't in the paper. I've reorganized the sentence slightly; hope that works.
